flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/HiveCatalog.java

```diff
@@ -590,6 +578,17 @@ private  static Table instantiateHiveTable(ObjectPath tablePath, CatalogBaseTabl
 		return hiveTable;
 	}
 
+	private static void setStorageFormat(StorageDescriptor sd, Map<String, String> properties) {
+		// TODO: simply use text format for now
+		String storageFormatName = DEFAULT_HIVE_TABLE_STORAGE_FORMAT;
+		StorageFormatDescriptor sfDescriptor = storageFormatFactory.get(storageFormatName);
+		checkArgument(sfDescriptor != null, "Unknown storage format " + storageFormatName);
```

Comment:
move input checks to the top?

Reply:
It doesn't directly check input argument. It checks whether we can get a StorageFormatDescriptor based on the input argument.